### PR TITLE
Upgrade dependency "trim"

### PR DESCRIPTION
**Pull request by mikeharder**

### CODE REVIEW

File: common/config/rush/pnpm-config.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+   * by Rush during installation.
+   *
+   * Order of precedence: `.pnpmfile.cjs` has the highest precedence, followed by
+   * `unsupportedPackageJsonSettings`, `globalPeerDependencyRules`, `globalPackageExtensions`,
+   * and `globalOverrides` has lowest precedence.
+   *
+   * PNPM documentation: https://pnpm.io/package_json#pnpmoverrides
+   */
+  "globalOverrides": {
+    "trim@0.0.1": "0.0.3"
```
Comment (mikeharder) [inline common/config/rush/pnpm-config.json:92]: This is the only line changed from the default config file.

File: common/config/rush/pnpm-lock.yaml [hunk 2]
```diff
@@ -6423,6 +6426,7 @@ packages:
 
   /@types/debug@4.1.8:
     resolution: {integrity: sha512-/vPO1EPOs306Cvhwv7KfVfYvOJqA/S/AXjaHQiJboCZzcNDb+TIJFN9/2C9DZ//ijSKWioNyUxD792QmDJ+HKQ==}
+    requiresBuild: true
```
Comment (mikeharder) [inline common/config/rush/pnpm-lock.yaml:6429]: These lines were added when I ran `rush update --recheck` with no other changes.  So they appear unrelated to my change, but I needed to run `rush update --recheck` to upgrade `trim`.

File: common/config/rush/.pnpmfile.cjs
Comment (mikeharder) [inline common/config/rush/.pnpmfile.cjs:1]: Default file generated by `rush init` with no changes.

File: rush.json [hunk 3]
```diff
@@ -7,13 +7,6 @@
   "rushVersion": "5.100.1",
   "pnpmVersion": "8.6.12",
   "ensureConsistentVersions": true,
-  /**
```
Comment (mikeharder) [inline rush.json:10]: Recent versions of `rush` moved `pnpm` options to file `pnpm-config.json`

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**